static_src/util/cf_api.js

```diff
@@ -198,8 +210,15 @@ export default {
     return this.fetchOne(`/spaces/${spaceGuid}/summary`);
   },
 
-  fetchSpaceEvents(spaceGuid) {
-    return this.fetchAllPages(`/spaces/${spaceGuid}/events`, results => results);
+  fetchSpaceEvents(spaceGuid, options) {
```

Comment:
Yep, valid point. The `q` parameter takes a handful of [values and operators](https://apidocs.cloudfoundry.org/245/spaces/list_all_events_for_the_space.html) for filtering, so I wanted to start moving our action API closer to something that gives us the flexibility offered by the CF api. Since other APIs also accept a `q` parameter I wanted to establish a convention rather than just sticking arbitrary second, third, etc arguments into these kinds of actions.

I'll have a stronger opinion once we start optimizing other APIs as well, but for now I'll let you make the call.